mslib/mscolab/models.py

```diff
@@ -216,5 +216,5 @@ def __init__(self, op_id, u_id, commit_hash, version_name=None, comment=None):
         self.op_id = int(op_id)
         self.u_id = int(u_id)
         self.commit_hash = str(commit_hash)
-        self.version_name = str(version_name)
-        self.comment = str(comment)
+        self.version_name = version_name
+        self.comment = comment
```

Comment:
The casts were introduced with the reasoning here: https://github.com/Open-MSS/MSS/pull/2470#discussion_r1721425806

We could keep them while still handling None properly, like so:
```suggestion
        if version_name is not None:
            self.version_name = str(version_name)
        if comment is not None:
            self.comment = str(comment)
```

This would also make more sense semantically, as it would use optional arguments only when they are supplied. Otherwise having the defaults on the columns is kind of useless.

What do you think?